docs/src/04_solution_strategy.adoc

```diff
@@ -26,9 +26,10 @@ We also have a Whatsapp community for the team, and a Notion wiki.
 === Important quality-related decisions
 
 |===
-|Quality attribute pursued|Solution chosen
-|Privacy|All stored password will be hashed, both client-side and server-side, to avoid password disclosure. The client-side password is also intended to prevent password discoverage in case it is a repeated one.|
-|Robustness|Currently, all validations will take place server-side to avoid not being properly taken care of due to JavaScript desactivation, such as when using the NoScript plug-in|
+|*Quality attribute pursued*|*Solution chosen*
+|Privacy|All stored password will be hashed, both client-side and server-side, to avoid password disclosure. The client-side password is also intended to prevent password discoverage in case it is a repeated one.
+|Robustness|Currently, all validations will take place server-side to avoid not being properly taken care of due to JavaScript desactivation, such as when using the NoScript plug-in
+|Speed|Since Wikidata has a 1 minute limit related to the API, the backend will start querying it upon start and fill the database with questions to increase speed and thus improve user experience|
 |===
 
 === Workflow
```

Comment:
I believe you are calling question availability as speed. Although they are related and both are required in the final product, probably speed should become a quantitative attribute sooner than later. On the other hand, I think the qualitative decision text is fine.
Also, we should look for more and better ways to implement privacy in the future.

Reply:
Regarding the first part, if I understand you correctly then replacing _speed_ for _availability_ should do it. Regarding second one, this is currently what we have regarding privacy, although I agree we should think better ideas (for instance, @Toto-hitori intends to drop HTTP and exclusively use HTTPS).

Reply:
About the first time, you are right. I also believe speed to be more of a quantitative than a qualitative attribute and we should fix its values within the following weeks.
Finally, I just wanted to remark the security as something to be careful about along the whole proyect.